### ISSUE TITLE
FAWE doesnt work, paper 1.13.2

### ISSUE THREAD

**Issue by Darkweasam** [IntellectualSites/FastAsyncWorldEdit-1.13#112]
# Bug report for FastAsyncWorldEdit 1.13.x

**Debug paste link**:

 https://incendo.org/paste/view/116af47feaae4716ac6530af4cd51992
**Description of the problem:**

FAWE doesnt work as intended using 1.13.2 paper latest(as of 8th of April)
Problems such as:
- Problem when you paste something (to different world)
- Brushes doesnt work at all
- Other minor problems, VoxelSniper doesnt seem to work correctly either
**How to replicate**:
 I believe its paper spigot and fawe doesnt want to work together

**Checklist**:

- [x] I included a `/fawe debugpaste` link https://incendo.org/paste/view/116af47feaae4716ac6530af4cd51992
- [x] I made sure there aren't duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of [FAWE for 1.13.x](https://ci.athion.net/job/FAWE-1.13/) Link is dead tho
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by N0tMyFaultOG** [comment]
Update [fawe](https://ci.athion.net/job/FastAsyncWorldEdit-Breaking/lastSuccessfulBuild/artifact/target/FastAsyncWorldEdit-bukkit-1.13.22.jar) and [favs](https://ci.athion.net/job/FastAsyncWorldEdit-Breaking/lastSuccessfulBuild/artifact/target/FastAsyncVoxelSniper-favs-1.13.22.jar)

**Comment by Tsoccerguy3** [comment]
//copy , //rotation , //paste = nothing in commit d7380781f411d6204a7adf257789d417d376ddd4  

don't use the build NOtMyFaultOG posted , wait for a fix

**Comment by N0tMyFaultOG** [comment]
> //copy , //rotation , //paste = nothing
> 
> there is a bug in commit [d738078](https://github.com/IntellectualSites/FastAsyncWorldEdit-1.13/commit/d7380781f411d6204a7adf257789d417d376ddd4)
> 
> don't use the build NOtMyFaultOG posted , wait for a fix

I'm unsure which line you are referencing your answer to, since it works with the build I linked and all following ones. If you tested, confirmed and reproduced an error with one of the commits being committed, feel free to raise an issue. 
Also, copy/pasting works perfectly fine btw, just to answer your question.
@Darkweasam Ignore my post above and gather yourself the latest build from [here](https://incendo.org/download) -> breaking. I just tested it and copy/pasting works fine, so does FAVS.
